plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/startup/AmazonQStartupActivity.kt

```diff
@@ -27,9 +40,43 @@ class AmazonQStartupActivity : ProjectActivity {
                 CodeWhispererExplorerActionManager.getInstance().setIsFirstRestartAfterQInstall(false)
             }
         }
-
+        startLsp(project)
```

Comment:
just want to make sure we're good if the indexing starts even if users are not logged in

Reply:
yes it is not relevant to login status